homeassistant/components/websocket_api/commands.py

```diff
@@ -778,7 +778,25 @@ async def handle_execute_script(
 
     context = connection.context(msg)
     script_obj = Script(hass, script_config, f"{const.DOMAIN} script", const.DOMAIN)
-    script_result = await script_obj.async_run(msg.get("variables"), context=context)
+    try:
+        script_result = await script_obj.async_run(
+            msg.get("variables"), context=context
+        )
+    except ServiceValidationError as err:
+        connection.logger.error(err)
+        connection.logger.debug("", exc_info=err)
+        connection.send_error(
+            msg["id"],
+            const.ERR_SERVICE_VALIDATION_ERROR,
+            str(err),
+            translation_domain=err.translation_domain,
+            translation_key=err.translation_key,
+            translation_placeholders=err.translation_placeholders,
+        )
+        return
+    except Exception as exc:  # pylint: disable=broad-except
+        connection.async_handle_exception(msg, exc)
+        return
```

Comment:
`except Exception` block can be removed as `async_handle_exception` will be called on any unhandled errors for any web socket command. 
@jbouwh Can you please remove it in a follow up PR?

Reply:
Sure

Reply:
https://github.com/home-assistant/core/pull/104727